package.json

```diff
@@ -2,46 +2,50 @@
   "name": "gulp-htmlmin",
   "description": "gulp plugin to minify HTML.",
   "version": "1.3.0",
-  "author": {
-    "name": "Jon Schlinkert",
-    "url": "https://github.com/jonschlinkert"
-  },
+  "homepage": "https://github.com/jonschlinkert/gulp-htmlmin",
+  "author": "Jon Schlinkert (https://github.com/jonschlinkert)",
   "contributors": [
     "Shinnosuke Watanabe <snnskwtnb@gmail.com> (https://github.com/shinnn)"
   ],
   "repository": "jonschlinkert/gulp-htmlmin",
+  "bugs": {
+    "url": "https://github.com/jonschlinkert/expand-files/issues"
+  },
   "license": "MIT",
-  "keywords": [
-    "format",
-    "gulp",
-    "gulpplugin",
-    "htm",
-    "html",
-    "minifier",
-    "minification",
-    "minify"
-  ],
   "files": [
     "index.js"
   ],
+  "main": "index.js",
```

Comment:
My linting tool adds this, I'm still tweaking it to making better decisions, I'll remove this field.

> I don't see why you use JSLint rather than ESLint.

I don't care which one we use. I'm planning on switching all of my projects to eslint. The only thing that matters to me is that whichever one we use actually works. Currently the script does not work (for me at least). If you want to keep it there that's fine as long as it works.